store/sqlstore/channel_store.go

```diff
@@ -1205,6 +1205,25 @@ func (s SqlChannelStore) GetMembers(channelId string, offset, limit int) store.S
 	})
 }
 
+func (s SqlChannelStore) GetChannelMembersTimezones(channelId string) store.StoreChannel {
+	return store.Do(func(result *store.StoreResult) {
+		var dbMembersTimezone []map[string]string
+		_, err := s.GetReplica().Select(&dbMembersTimezone,
+			`SELECT
+					Users.Timezone
```

Comment:
nit: indention here is weird. I'd propose starting the backtick on the line above, and then keeping things consistent:
```golang
		if _, err := s.GetReplica().Select(&dbMembersTimezone, `
                        SELECT
                                  Users.Timezone
                        FROM
                                  // ...
  		`, map[string]interface{
                         "ChannelId": channelId,
                 }); err != nil {
                      // ...
                 }

Reply:
why here not use the idiomatic err handling?

Reply:
Hmm, the above is idiomatic: see https://github.com/mattermost/mattermost-plugin-demo/pull/8#discussion_r221970808 for a more detailed explanation.